### DISCUSSION TITLE
How to use Cordova plugins in capacitor with ionic react

### DISCUSSION THREAD

**Discussion by JeelRajodiya** [ionic-team/capacitor#4009] [Q&A] [answered]
I wanted to change the colour of navigation bar in android app that is made by capacitor but I can't find any plugins that give me control over the navbar so I Found Cordova plugins but I don't know how do I use them in my ionic project here's what i tried

npm install cordova-plugin-navigation-bar
npx cap sync

Now I don't know what to do after this.
Some questions about it:
How do I import this Cordova plugin in my app.tsx file.
Is there any alternative plugin available in ionic

The goal of the plugin should be getting control over the android navigation bar (by control I mean changing its colour hiding it etc.)

**Comment by imhoffd** [comment, marked as answer]
@jeel2331 Please follow [this proposal](https://github.com/capacitor-community/proposals/issues/51) for a Navigation Bar plugin. You'll find a compatible Cordova plugin listed in it.
